### ISSUE TITLE
Email link inactive on Terms of Use screen

### ISSUE THREAD

**Issue by shaunan** [StatCan/StatcanPublicationUAT#593]
On the Terms of Use screen, the email link “infostats@statscan.gc.ca” is not active. Users cannot click or select the link to open their email client to create an email.

The email link should be active, using easy to understand link text. 

![GitHub6](https://github.com/StatCan/StatcanPublicationUAT/assets/115822055/7264021d-57b0-4e57-9b0d-0e4f2ff7178a)

**Comment by weglmar** [comment]
#578
